### PR TITLE
Add a link to the project status page on CircleCI

### PR DESCRIPTION
**Pull request by tnir**
- ~~Avoids to use shields.io's badge~~
- Adds a useful link to the CircleCI pipeline results

cf. some usecase in a certain project: wbernest/mattermost-plugin-rssfeed#18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Plugin Starter Template ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-starter-template/master.svg)
+# Plugin Starter Template [![](https://circleci.com/gh/mattermost/mattermost-plugin-starter-template.svg?style=shield)](https://circleci.com/gh/mattermost/mattermost-plugin-starter-template)
```
Comment (hanzei) [inline README.md]: I like the addition of the link to circleci.

What is your reasoning for not fetching the icon from shields.io? I find it quite handy to get all icons from the same source in the same style.

Comment (tnir) [reply README.md]: While I am really sure the design can be unified with shield.io, I do not prefer shield.io badges due to its stability. e.g., badges/shields#1518 for Code Climate case.

Also  can be acceptable to me.

Comment (hanzei) [reply README.md]: Then let's just stick with the icon from shield.io. I like the unified source. Would you be fine with this?

**Comment by hanzei** [comment]
@tnir If you don't mind I will do the proposed changes and then merge.

File: README.md (hunk 1 above)
Comment (tnir) [reply README.md]: @hanzei Thanks. Do you mean you like the following?

```suggestion
# Plugin Starter Template [![](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-starter-template/master.svg)](https://circleci.com/gh/mattermost/mattermost-plugin-starter-template)
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Plugin Starter Template ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-starter-template/master.svg)
+# Plugin Starter Template [![](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-starter-template/master.svg)](https://circleci.com/gh/mattermost/mattermost-plugin-starter-template)
```
Comment (hanzei) [inline README.md]:
```suggestion
# Plugin Starter Template [![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-starter-template/master.svg)](https://circleci.com/gh/mattermost/mattermost-plugin-starter-template)
```

**Comment by tnir** [comment]
@hanzei Thanks for the comment. Adpated thr suggestion.

**Comment by ali-farooq0** [comment]
@tnir if you merge `master` into your branch that should fix the ci error

**Comment by moussetc** [comment]
Super ! 



Télécharger BlueMail pour Android



Le 20 sept. 2019 à 04:14, à 04:14, Ali Farooq <notifications@github.com> a écrit:

**Comment by tnir** [comment]
Thanks @ali-farooq0 

Was able to confirm all tests on CI passed at https://github.com/mattermost/mattermost-plugin-starter-template/runs/229404880

**Pull request merged**